templates/CRM/Price/Form/Calculate.tpl

```diff
@@ -150,7 +150,8 @@ function calculateSelectLineItemValue(priceElement) {
  */
 function calculateText(priceElement) {
   //CRM-16034 - comma acts as decimal in price set text pricing
-  var textval = parseFloat(cj(priceElement).val().replace(thousandMarker, ''));
+  //CRM-19937 - dollar sign easy mistake to make by users.
+  var textval = parseFloat(cj(priceElement).val().replace(thousandMarker, '').replace('$', ''));
```

Comment:
@jmcclelland I see we already have currency symbol in a `symbol` variable as we have for `thousandMarker`. We should be able to replace the same using:

    .replace(thousandMarker, '').replace(symbol, ''));

Can you please try this and check if it works for you ?

Thanks.